app/(saas)/settings/billing-profiles/[id]/coworkers/_features/update-coworker-role/update-coworker-role.tsx

```
@@ -4,6 +4,7 @@ import { BillingProfileReactQueryAdapter } from "@/core/application/react-query-
 import { BillingProfileRole } from "@/core/domain/billing-profile/billing-profile.types";
 
 import { Select, SelectContent, SelectItem, SelectTrigger, SelectValue } from "@/shared/ui/select";
+import { TypographyP } from "@/shared/ui/typography";
```

ESLint

[error] 'TypographyP' is defined but never used.

(unused-imports/no-unused-imports)